__tests__/actionCreators.test.js

```diff
@@ -35,3 +37,9 @@ describe('signedOut', () => {
     expect(signedOut().type).toEqual('SIGN_OUT_SUCCESS')
   })
 })
+
+describe('update', () => {
+  it('returns a function', () => {
+    expect(update()).toBeInstanceOf(Function)
```

Comment:
would we not want a test that is a bit more useful than this?  Like, showing what the function does?

Reply:
I've added a test, but it's sort of a mess of async mocks.  I'm not sure if it's more helpful.

Reply:
I see what you mean.